console/frontend/src/main/frontend/src/app/app.component.ts

```diff
@@ -565,6 +578,15 @@ export class AppComponent implements OnInit, OnDestroy {
     }
   }
 
+  hasAdapterReloaded(adapter: Adapter): boolean {
+    const oldAdapter =
+      this.appService.adapters[`${adapter.configuration}/${adapter.name}`];
+    if (adapter.upSince > oldAdapter?.upSince) {
```

Comment:
It's possible to return this condition instead of true or false?

Reply:
You're correct, it was a void function before with a loop for all adapters.
Didn't think it more through after I changed it to this